packages/app/components/UsersTabContent.tsx

```diff
@@ -0,0 +1,188 @@
+    marginTop: "15px"
+});
+
+const Table = styled("table", {
+    border: "1px solid black",
+    textAlign: "center"
+});
+
+const TableHeader = styled("th", {
+    border: "1px solid black",
+    padding: "10px"
+});
+
+const TableData = styled("td", {
+    border: "1px solid black",
+    padding: "10px"
+});
+
+const UsersTabContent = () => {
+
+    const [mAddresses, setMAddresses] = useState<string[]>([]);
+    const [membershipDetails, setMembershipDetails] = useState<MembershipDetailMap>({});
+    const [users, setUsers] = useState<{[key: string]: number}>({});
+    const [showLoading, setShowLoading] = useState<boolean>(false);
+    const address = useAddress();
+    const web3 = useWeb3();
+    const chainId = useChainId();
+    const [selectedOption, setSelectedOption] = useState<string>("");
+    const contractInstance = useMemo<Contract>(() => {
+        const contract = new web3.eth.Contract(getAbiFromJson(passportFactoryJson));
+        contract.options.address =
+        contractAddressesByNetworkId[chainId]?.passportFactory || "";
+        return contract;
+    }, [web3, chainId]);
+
+    useEffect(() => {
+        // Get all memberships assosciated with wallet
+        if (contractInstance.options.address) {
+            setShowLoading(true);
+            (contractInstance.methods.getMemberships() as ContractSendMethod)
+            .call({
+                from: address,
+            })
+            .then((r: string[]) => {setMAddresses(r)})
+            .catch(console.error)
+            .finally(() => setShowLoading(false));
+        }
+    }, [contractInstance, address, setMAddresses]);
+
+    useEffect(() => {
+        if(selectedOption){
+            setShowLoading(true);
+            const contract = new web3.eth.Contract(getAbiFromJson(passportJson));
+            contract.options.address = selectedOption;
+            // Get tokenIds which have been bought
+            contract.getPastEvents('Transfer', {
+                filter: {
+                    _from: '0x0000000000000000000000000000000000000000'
+                },
+                fromBlock: 0
+            }).then((events) => {
+                const tokenIds = events.map((event) => event.returnValues.tokenId)
+                return tokenIds;
+            }).then((tokenIds) => {
+                // Get the owner of each of these bought tokens
+                const ownerPromises = tokenIds.map((tokenId) => {
+                    return contract.methods.ownerOf(tokenId).call();
+                });
+                Promise.all(ownerPromises).then((ownerAddrs) => {
+                    const owners: {[key: string]: number} = {};
+                    ownerAddrs.forEach((ownerAddr) => {
+                        owners[ownerAddr] = (owners[ownerAddr]+1) || 1;
+                    });
+                    setUsers(owners);
+                })
+                .finally(() => setShowLoading(false));
+            });
+        }
+    }, [selectedOption, setUsers]);
```

Comment:
![image](https://user-images.githubusercontent.com/7143571/151685270-5beca667-1f34-4156-8511-ac2c743ac5b6.png)

Seems like there was another lint warning and a swc failure. I'm not sure why the swc failure would happen, but could we fix these lint warnings and see it's related?